### ISSUE TITLE
seneca.log should not modify passed parameters

### ISSUE THREAD

**Issue by StarpTech** [senecajs/seneca#535]
Hi, is this the correct behaviour?
```js
let obj = { success: false, error: new Error('user not found') }
this.log.info(obj)
next(null, obj);

console.log(obj)

/**
{
success: false,
error: new Error('user not found')
actidi: ...
when: ...
plugin_name.....
......
}
*/

```

**Comment by StarpTech** [comment]
Duplicate https://github.com/senecajs/seneca/issues/531

**Comment by dgonzalez** [comment]
You might want to have a look to https://github.com/senecajs/seneca-log-filter. Not fully documented yet but ping me if you need a hand. Cheers!

**Comment by mcdonnelldean** [comment]
Merged with #531
